.github/workflows/build-release.yml

```diff
@@ -18,29 +20,34 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Set date
-        id: date
-        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
-      - name: Set tag
-        id: tag
-        run: if [ "${GITHUB_REF#refs/*/}" == "main" ]; then echo ::set-output name=branch::latest; else echo ::set-output name=branch::${GITHUB_REF#refs/*/}; fi
-      - name: Dockerhub login
-        env:
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-        run: |
-          echo "${DOCKER_PASSWORD}" | docker login --username ${DOCKER_USERNAME} --password-stdin
+
+      # Docker Image 构建环境
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v2
+
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      - name: Build dockerfile (with push)
-        env:
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      # ReleaseTag
+      - name: Set Version
+        id: set-version
         run: |
-          docker buildx build \
-          --platform=linux/amd64,linux/arm/v7,linux/arm64 \
-          --output "type=image,push=true" \
-          --file ./Dockerfile . \
-          --tag $(echo "${DOCKER_USERNAME}" | tr '[:upper:]' '[:lower:]')/ttrss:${{ steps.tag.outputs.branch }} \
-          --tag $(echo "${DOCKER_USERNAME}" | tr '[:upper:]' '[:lower:]')/ttrss:${{ steps.tag.outputs.branch }}-${{ steps.date.outputs.date }}
+          echo "version=`date +'%y-%m-%d'`" >> $GITHUB_ENV
+          
+      # Build and push
+      - name: Build dockerfile
+        uses: docker/build-push-action@v4
+        with:
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          # platforms: linux/amd64
+          push: true
+          tags: |
+            wangqiru/ttrss:stable-${{ env.version }}
```

Comment:
Adding an undocumented tag may confuse users. 

It is suggested to maintain consistency with the current tagging convention, which includes a `latest` tag and a dated `latest` tag format such as `latest-2023-01-23`.